### ISSUE TITLE
Need TOS page in Login

### ISSUE THREAD

**Issue by mattkasner** [somasole/SomaSole#92]
The user must ok the terms of service that we will provide in order to use the app. 

I will the full TOS sheet from the lawyers on Monday

**Comment by mrigdon-zz** [comment]
It'd be nice to have a mockup of this as well.

**Comment by mattkasner** [comment]
[TOS.pdf](https://github.com/mrigdon/SomaSole/files/357832/TOS.pdf)

**Comment by mrigdon-zz** [comment]
Where do you want this inserted in the create account process?

**Comment by mattkasner** [comment]
After they create their account and before they enter the app for the first
time.

**Comment by mrigdon-zz** [comment]
Do you have the full sheet?

**Comment by mattkasner** [comment]
Still waiting on Lawyer

**Comment by mrigdon-zz** [comment]
@mattkasner how we doing on this?

**Comment by mattkasner** [comment]
@mrigdon [SomaSole Inc - Terms of Use (2).docx](https://github.com/mrigdon/SomaSole/files/376593/SomaSole.Inc.-.Terms.of.Use.2.docx)

**Comment by mattkasner** [comment]
@mrigdon I attached the UI awhile back and just uploaded the information that will
be included as the Terms of Service
